msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -428,6 +428,8 @@ async def on_teams_members_added_activity(
 
 The `teamMemberRemoved` event is sent to your bot if it is removed from a team and every time any user is removed from a team that your bot is a member of. You can determine if the new member removed was the bot itself or a user by looking at the `Activity` object of the `turnContext`.  If the `Id` field of the `MembersRemoved` object is the same as the `Id` field of the `Recipient` object, then the member removed is the bot, otherwise it is a user.  The bot's `Id` will generally be: `28:<MicrosoftAppId>`
 
+[!Note] When a user is removed from a tenant, `membersRemoved converstaionUpdate` event is triggered.
```

Comment:
@v-rodriguezd Typo: "conversationUpdate"
Also we should specify that the event is sent when the user is permanently deleted.